File: tmp-mei/fgen4255027576711118260/base/flows/_OnePerStepSinks_0.java
package base.flows;
public interface _OnePerStepSinks_0 extends base.flows._SinkDecorator_0{
_OnePerStepSinks_0 $self = new _OnePerStepSinks_0Impl();
base.flows._Sink_1 $hash$imm(base.flows._Sink_1 sink_m$);
static base.flows._Sink_1 $hash$imm$fun(base.flows._Sink_1 sink_m$, base.flows._OnePerStepSinks_0 $this) {
  return ((base.flows._Sink_1)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.flows._Sink_1) null;
}})
);
}
}

File: tmp-mei/fgenjs5030319945734616766/base/flows/_OnePerStepSinks_0.js
import { base$$flows$$_SinkDecorator_0 } from "../../base/flows/_SinkDecorator_0.js";
import * as rt from "../rt/main.js";

export class base$$flows$$_OnePerStepSinks_0 extends base$$flows$$_SinkDecorator_0 {
  static $self = new base$$flows$$_OnePerStepSinks_0();

  $hash$imm(sink_m$) {
  return base$$flows$$_OnePerStepSinks_0.$hash$imm$fun(sink_m$, this);
}

  static $hash$imm$fun(sink_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}
